tests/Domain/Integrations/Controllers/IntegrationControllerTest.php

```diff
@@ -113,6 +120,35 @@ private function givenThereIsAnIntegration(): Integration
         return $integration;
     }
 
+    private function givenTheActingUserIsAContactOnIntegration(Integration $integration): Contact
+    {
+        $user = Auth::user();
+
+        if (!$user) {
```

Comment:
I think the `$user` should never be `null`?

Reply:
The inferred return type of `Auth::user()` is `UserModel|Authenticatable|null`